web/packages/teleport/src/Console/DocumentSsh/Terminal/Terminal.tsx

```diff
@@ -73,6 +74,10 @@ export const Terminal = forwardRef<TerminalRef, TerminalProps>((props, ref) => {
     termCtrl.open();
 
     const { unregister } = termCtrl.registerCustomKeyEventHandler(event => {
+      if (props.disableCtrlC && event.ctrlKey && event.key == 'c') {
```

Comment:
99% of the time we want to use [the strict equality operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality).
```suggestion
      if (props.disableCtrlC && event.ctrlKey && event.key === 'c') {
```